Classes/Command/NodeCommandController.php

```diff
@@ -516,6 +517,7 @@ private function searchTermMatch($term, $value)
      * @param NodeInterface[]|array[] $nodes
      * @param string $indend
      * @param string $propertyName which property to display, defaults to title
+     * @throws \Neos\ContentRepository\Exception\NodeException
```

Comment:
I will not mix this kind-of stuff in this PR. This PR should only address the upgrade to new Flow/Neos versions and not also do the cleanups to make the new phpStorm defaults happy. This has nothing to do with the upgrade itself.

Reply:
in the meantime, just disable the warning in phpstorm, e.g. using:

<img width="914" alt="image" src="https://user-images.githubusercontent.com/6108134/39622945-40eb6898-4f94-11e8-8bef-7cffab9ae914.png">